tensorflow_estimator/python/estimator/tpu/tpu_context.py

```diff
@@ -795,18 +795,19 @@ def __init__(self, replica_id_in_sync):
     # pylint: enable=protected-access
 
   def __enter__(self):
+    ctx = eager_contex.context()
```

Comment:
You have a typo here - should be eager_context

Reply:
Fixed, thanks for pointing it out